app/services/search_form_param_parser.rb

```diff
@@ -41,4 +44,18 @@ def filter_keys
     EvidenceSummarySearchForm.checkbox_filters + 
       EvidenceSummarySearchForm.radio_button_filters
   end
+
+  def years_array(option)
+    to_range[option].map do |n|
+      n.year.ago.strftime("%Y").to_i
+    end
+  end
+
+  def to_range
```

Comment:
I think the name of the method could be renamed to `years_range` because when I read the code above `to_range` makes me thing that a `Range` object will be return which is not the case.